manifests/params.pp

```diff
@@ -1,7 +1,7 @@
 class unicorn::params {
 
   case $::operatingsystem {
-    'debian', 'ubuntu': {
+    'debian', 'ubuntu', 'centos': {
```

Comment:
The way this was written was funky, but I don't _think_ you want 'centos' here.  With centos here, this will change the behavior on centos 6 vs centos 7, which I don't think is supposed to apply to centos, probably only debian and ubuntu(which makes sense, ubuntu is based on Debian)  Likewise, I _think_ you could remove the 'ubuntu' section below, as this is going to match of those here